doc/user-guides/gateway-dns.md

```diff
@@ -0,0 +1,224 @@
+# Gateway DNS for Cluster Operators
+
+This user guide walks you through an example of how to configure DNS for all routes attached to an ingress gateway.
+
+<br/>
+
+## Requisites
+
+- [Docker](https://docker.io)
+- [Rout53 Hosted Zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html)
+
+### Setup
+
+This step uses tooling from the Kuadrant Operator component to create a containerized Kubernetes server locally using [Kind](https://kind.sigs.k8s.io),
+where it installs Istio, Kubernetes Gateway API and Kuadrant itself.
+
+Clone the project:
+
+```shell
+git clone https://github.com/Kuadrant/kuadrant-operator && cd kuadrant-operator
+```
+
+Setup the environment:
+
+```shell
+make local-setup
+```
+
+Install metallb:
+```shell
+make install-metallb
+```
+
+Fetch the current kind networks subnet:
+```shell
+docker network inspect kind -f '{{ (index .IPAM.Config 0).Subnet }}'
+```
+Response:
+```shell
+"172.18.0.0/16"
+```
+
+Create IPAddressPool within kind network(Fetched by the command above) e.g. 172.18.200
+```shell
+kubectl -n metallb-system apply -f -<<EOF
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: kuadrant-local
+spec:
+  addresses:
+  - 172.18.200.0/24
+---
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: empty
+EOF
+```
```

Comment:
I think this would be better scripted and part of the default local setup (`make local-setup`) I'd say most user guides, docs, demos etc.. would be better for it since you won't need to do stuff like this https://github.com/Kuadrant/kuadrant-operator/blob/main/doc/user-guides/gateway-rl-for-cluster-operators.md?plain=1#L50 and forward ports to your host. 

@guicassolato @eguzki Any thoughts on that? Don't mind doing it, just don't want to waste my time if there is a reason not to.

Reply:
do we do this in MGC also? If not seems it would be useful there too?

Reply:
We, for reasons i don't know or can't remember, create an entirely different docker network in MGC and so know what the subnet is and create the IPAddressPool with that. This is more like what the kind docs tell you to do with the existing kind docker network https://kind.sigs.k8s.io/docs/user/loadbalancer/

@sergioifg94 Do you know if there was a reason we created a different docker network rather than just using the one that was already being created?

Reply:
Yes, the reason for it is that if you don't create one and let it automatically create, it it will assign the next free address, so the addresses could differ depending on what local networks people might already have. Creating one beforehand ensures that the addresses are consistent for everyone